.gitignore

```diff
@@ -8,12 +8,37 @@
 *.so
 *.so.*
 
+#map files
+*.map
+
 #generated proxy
 *_u.c
 *_u.h
 *_t.c
 *_t.h
 
 #output files
+output/
```

Comment:
Adding file/folder paths in .gitignore doesn't seem an extensible approach. We may change the layout of the generated files, folders where temp files are put, etc.   Then it will require to change .gitinore file as well.  

We'd like to just have a simple rule for .gitinore which just ignore some specific file (with specific extensions), but not specify the path.